src/common/utility.c

```diff
@@ -26,6 +26,13 @@
 #include <unistd.h>
 #endif
 
+#if defined _WIN32 
+#include <stdlib.h>
```

Comment:
These headers were not in the original source, and I tried to avoid adding headers which might interfere with the existin Linux or OSX build. `Math.h `was needed because of `modff`, `fabs `functions, and `gi18n.h` was needed for `N_` and `_` . Do you think if I removed the conditional that would cause no problem on Linux build?